changelogs/fragments/5772-consul-deprecate-params-when-absent.yml

```diff
@@ -0,0 +1,2 @@
+deprecated_features:
+  - consul - deprecate using parameters unused for state=absent (https://github.com/ansible-collections/community.general/pull/5772).
```

Comment:
Because that is holding back implementing a `required_if` and a `mutually_exclusive` - see the long comment in the code about what to do when removing in c.g. 8.0.0